src/sql/src/plan.rs

```diff
@@ -655,6 +655,7 @@ pub struct Ingestion {
     pub desc: SourceDesc,
     pub source_imports: BTreeSet<GlobalId>,
     pub subsource_exports: BTreeMap<GlobalId, usize>,
+    pub progress_subsource: Option<GlobalId>,
```

Comment:
Some parts call this `remap_collection`, others call it `progress_subsource`. I know the former is more the internal name and the latter is the user-facing concept. So could be fine.

Reply:
Yes, I tried to make this division along the same lines everywhere; there are a number of places where we make this pivot between the external and internal terms, all along the same "fault line." This is a somewhat arbitrary style that I don't have an attachment to so would be glad to make a change if you recommend something else.

Reply:
Looks good for now! 👌